catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -650,10 +650,12 @@ private InsertResultType getInsertResultFromResponse(CreateResponse createRespon
     private int deleteRecords(DeleteAction deleteAction)
             throws CswException, FederationException, IngestException, SourceUnavailableException,
             UnsupportedQueryException {
-        List<QName> qNames = typeStringToQNames(deleteAction.getTypeName(),
-                deleteAction.getPrefixToUriMappings());
 
-        QueryRequest queryRequest = queryFactory.getQuery(deleteAction.getConstraint(), qNames);
+        QueryRequest queryRequest = queryFactory.getQuery(deleteAction.getConstraint());
+        if (!CswConstants.CSW_RECORD.equals(deleteAction.getTypeName())) {
```

Comment:
It was because we were treating csw:Record as the default value so it would be using the default tag of resource. However now that I am looking at it it would be simpler to remove the 'if' logic and update the blueprint configuration for this case instead. That way we can always apply the tags logic.